cpp/Platform.Equality.Tests/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.8)
+project(Platform.Equality)
+
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+
+include_directories(../Platform.Equality)
```

Comment:
here it is better to do the following:
in the root / cpp put another CMakeListst.txt with the following content:
```
add_subdirectory (Platform.Equality)
add_subdirectory (Platform.Equality.Tests)
```
And here do
```
target_link_libraries(Platform.Equality.Tests Platform.Equality gtest gtest_main)
```

Reply:
We can try to put in `cpp` directory, and we will see how it goes.